### DISCUSSION TITLE
How to merge a pull request without reviewing the code ?

### DISCUSSION THREAD

**Discussion by xiaowoonly** [community/community#38626] [New to GitHub] [answered]
to get the github YOLO badge

**Comment by eternitywen** [comment, marked as answer]
- Create a new pull request in your repo and open it
- In the top right corner at Reviewers section, request someone to review your pull request
- Merge the pull request without finishing the review.

**Reply by abbyenredes** [reply]
Thanks, it worked for me

**Comment by coderbaba0** [comment]
- Create a new pull request in your repo and open it
- In the top right corner at Reviewers section, request someone to review your pull request
- Merge the pull request without finishing the review.

**Comment by Nishanth-Siraj** [comment]
- Create a new pull request in your repo and open it
- In the top right corner at Reviewers section, request someone to review your pull request
- Merge the pull request without finishing the review.

**Comment by Sreejithk1996** [comment]
1. Create a new pull request in your repo and open it

2. In the top right corner at Reviewers section, request someone to review your pull request

3. Merge the pull request without finishing the review.

**Comment by ali-workshop** [comment]
Create a new pull request in your repo and open it

In the top right corner at Reviewers section, request someone to review your pull request

Merge the pull request without finishing the review.

**Comment by BaybyPig0329** [comment]
Create a new pull request in your repo and open it

In the top right corner at Reviewers section, request someone to review your pull request

Merge the pull request without finishing the review.

**Comment by davidaMeteo** [comment]
Create a new pull request in your repo and open it

In the top right corner at Reviewers section, request someone to review your pull request

Merge the pull request without finishing the review.

**Comment by bumblebro** [comment]
Create a new pull request in your repo and open it

In the top right corner at Reviewers section, request someone to review your pull request

Merge the pull request without finishing the review.

**Comment by piyuminadee** [comment]
Create a new pull request in your repo and open it
In the top right corner at Reviewers section, request someone to review your pull request
Merge the pull request without finishing the review.

**Comment by Omni-HuB** [comment]
1. Create a new pull request in your repo and open it.
2. In the top right corner in the Reviewers section, request someone to review your pull request.
3. Merge the pull request without finishing the review.

**Comment by CodeInfiltrator** [comment]
First create a new pull request in your own repo and open it and then in the top right corner in the reviewers section, request someone to review your pull request, last merge the pull request without finishing the review.

**Comment by SuvasishChowdhury** [comment]
When you create a pull request from your any repository then in the top of the right panel or right side of your window you will find reviewers section.
Also you will see Request button of Reviewers block.
Then request someone to review this code.
Next don't wait for him/her to finish review just merge your pull request.
You will find this badge appear it on your Achievement Dashboard :)

**Reply by Chaitrali-tambe** [reply]
This helped me thanks

**Reply by SuvasishChowdhury** [reply]
My pleasure!

**Comment by TGouriSankar** [comment]
1.     Create a new pull request in your repo and open it
2.     In the top right corner at Reviewers section, request someone to review your pull request
3.     Merge the pull request without finishing the review.

**Comment by shopinstyle2020** [comment]
> - Create a new pull request in your repo and open it
> - In the top right corner at Reviewers section, request someone to review your pull request
> - Merge the pull request without finishing the review.

**Comment by shopinstyle2020** [comment]
> - Create a new pull request in your repo and open it
> - In the top right corner at Reviewers section, request someone to review your pull request
> - Merge the pull request without finishing the review.

**Comment by andrijajovanovic98** [comment]
just create a new pull request in your repo and open it after that 
In the right corner in the Reviewers section, request someone to review your pull request and 
merge the pull request without finishing the review :)

**Comment by ritvak** [comment]
1. Create a pull request in your repository and open it. 

2. In the top-right corner, under the Reviewers section, assign someone to review your pull request. 

3. Proceed to merge the pull request without waiting for the review to be completed.

**Comment by dinildamsith** [comment]
### ✅ Steps to Merge a Pull Request Without Review

1. **Create a Pull Request**:

   * Navigate to your repository on GitHub.
   * Click on the **"Pull requests"** tab.
   * Click **"New pull request"**, select the branches to merge, and click **"Create pull request"**.([[Graphite.dev](https://graphite.dev/guides/prevent-merge-without-review-github?utm_source=chatgpt.com)][1])

2. **Request a Reviewer**:

   * On the right sidebar of the pull request page, find the **"Reviewers"** section.
   * Click **"Request"** and select any collaborator or yourself as the reviewer.([[GitHub](https://github.com/orgs/community/discussions/38626?utm_source=chatgpt.com)][2])

3. **Merge Without Waiting for Review**:

   * Scroll down to the bottom of the pull request.
   * Click the **"Merge pull request"** button.
   * Confirm the merge when prompted.([[GitHub Docs](https://docs.github.com/articles/merging-a-pull-request?utm_source=chatgpt.com)][3])

By merging the pull request without completing the review, you fulfill the criteria for the YOLO badge.([[GitHub](https://github.com/orgs/community/discussions/38626?utm_source=chatgpt.com)][2])

---

### 🏆 About the YOLO Badge

The **YOLO** badge is awarded when you merge a pull request without any code review. It's a playful nod to merging without the usual safeguards.

---

### ⚠️ Important Considerations

* **Branch Protection Rules**: If your repository has branch protection rules that require reviews before merging, you'll need to adjust these settings:

  * Go to **Settings** > **Branches**.
  * Under **Branch protection rules**, edit or remove rules that require pull request reviews.([[Graphite.dev](https://graphite.dev/guides/prevent-merge-without-review-github?utm_source=chatgpt.com)][1])

* **Repository Permissions**: Ensure you have the necessary permissions to merge pull requests in the repository.

---

For more details and community discussions on this topic, you can refer to the following resources:

* GitHub Community Discussion: [[How to merge a pull request without reviewing the code?](https://github.com/orgs/community/discussions/38626)](https://github.com/orgs/community/discussions/38626)
* Stack Overflow Thread: [[How to merge a pull request without reviewing the code?](https://stackoverflow.com/questions/72662385/how-to-merge-a-pull-request-without-reviewing-the-code-to-get-the-github-yolo)](https://stackoverflow.com/questions/72662385/how-to-merge-a-pull-request-without-reviewing-the-code-to-get-the-github-yolo)

Let me know if you need further assistance!
